### ISSUE TITLE
model save 시 size 가 0B로 나옵니다.

### ISSUE THREAD

**Issue by DevKiHyun** [AiHackathon2018/AI-Vision#97]
```
def bind_model(model):
    def save(dir_name):
        print("dir name:", dir_name)
        saver = tf.train.Saver()
        os.makedirs(dir_name, exist_ok=True)
        saver.save(model.sess, dir_name)
    nsml.bind(save=save, load=load, infer=infer)

model.network()
model.optimize()
model.predict()
model.name = 0
bind_model(model) # First model is named '0'
'''
Session
'''
tf_config = tf.ConfigProto()
tf_config.gpu_options.allow_growth = True
model.sess = tf.Session(config=tf_config)

bTrainmode = False
if config.mode == 'train':
     bTrainmode = True
     model.sess.run(tf.global_variables_initializer())

     for epoch in range(nb_epoch):
           nsml.save("{}_{}".format(model.name, epoch))
```

 Resource_exhausted/ir_ph1_v2/94

Checkpoint    Last Modified    Elapsed    Summary                                                                    Size
------------  ---------------  ---------  -------------------------------------------------------------------------  ------
0_0           11 minutes ago   11.499     epoch=0, loss=7.202867572372026, epoch_total=5, acc=0.0009853603897263875  0 B
0_1           10 minutes ago   94.056     epoch=1, loss=7.202867565928281, epoch_total=5, acc=0.0009853603897263875  0 B

이렇게 학습 for문 마다 save를 시켜주면 save 되서 model ls 로 표시하면 checkpoint도 표시가 됩니다. 근데 마지막에 size 부분이 0B가 뜹니다. 모델 변수들이 저장이 안되었다는 얘기 아닌가요? 연관되는 부분인지는 모르겠지만 baseline의 코드는 loss, acc 의 변화가 있는데 제 모델로 동일한 트레이닝 set을 넣고 돌리면 loss, acc가 변동이 없습니다. 로컬의 imagenet 데이터셋에 대해선 정상적으로 학습이 되는것도 테스트 해봤습니다. 적어도 baseline 정도 처럼의 loss, acc의 변동이 있어야 될텐데 전혀 미동을 안합니다. gpu 사용량을 있는거 보면 연산 자체는 하는 것 같습니다만...

**Comment by Hackoperation** [comment]
안녕하세요,

위의 코드를 봤을때 save하실때 디렉토리를만들고 디렉토리에 save를 해서 문제인것같습니다. `saver.save(model.sess, dir_name)`
위부분을 파일에 저장하는것으로 바꿔주시면 될것같습니다.

감사합니다 :)
